product_docs/docs/biganimal/release/getting_started/managing_cluster.mdx

```diff
@@ -38,4 +38,4 @@ You can enable in-app inbox or email notifications to get alerted when the pause
 
 !!!note
```

Comment:
Space between '!!!' and 'note'?

Reply:
Not needed, although it looks nicer. If you're using indented admonitions, putting a space there increases the necessary indentation by 1, which can be ambiguous with code formatting. For fenced admonitions, whichever style you prefer.

Reply:
https://github.com/EnterpriseDB/docs/blob/develop/README.md#admonitions-notes-warnings-etc

!!! space type space title string.

This is as previously documented. If there's a wierd side effect where spaces on preceding the type on indented admonitions influence the subsequent count of spaces for the following block, we should log that as a bug because its kinda bonkers :)

Oh and looking over a whole bunch of admonition implementations, !!! space type space stringtitle seems to be the common structure.